build-tools/packages/build-cli/src/library/releasePrepChecks.ts

```diff
@@ -135,7 +135,7 @@ export const CheckHasRemoteBranchUpToDate: CheckFunction = async (
 		return {
 			message: `Error when checking remote branch. Does the remote branch exist? Full error message:\n${
```

Comment:
If this is the error case I was hitting, we might want to consider making failing this check non fatal (so you can choose to continue, or it continues with a warning, or lets you know what flag to use to skip this), and also it seems odd there is no way to just skip this single check with the CLI flags, having to skip all checks instead is odd when other checks have opt outs.

Seems like the allChecks map we have in prepare.ts should probably include the cli flag for each check (or use the function name) so we can make more logic general instead of per check (ex: when reporting all checks that fail, automatically include the flag to send to opt out of the check)

Reply:
Agreed in principle, but I think that level of change is out of scope for now. I just want to fix the blocking bugs.
I'll follow up with the build-tools owner and let them prioritize.